main.py

```diff
@@ -49,4 +49,4 @@ def main():
     
 
 if __name__ == '__main__':
-    main()
+    main()
```

Comment:
This looks like an unnecessary, accidental, whitespace-only change. Please avoid this as it clutters the diff and causes confusion.